liri.js

```diff
@@ -0,0 +1,136 @@
+var keys = require("./keys.js");
+var request = require("request");
+var Twitter = require("twitter");
+var twitter = new Twitter(keys.twitKeys);
+var Spotify = require('node-spotify-api');
+var spot = new Spotify(keys.spotKeys);
+var fs = require("fs");
+var command = process.argv[2];
+var input = process.argv.slice(3);
+
+if (input.length !== 0) input = addSpaces(input);
+
+
+if (input.length !== 0) {
+  doSomething(command, input);
+} else {
+  doSomething(command);
+}
+
+function doSomething(task, argument = "") {
+  write(task + " " + argument);
+  switch (task) {
+    case "my-tweets":
+      if (argument !== "") {
+        myTweets(argument);
+      } else {
+        myTweets();
+      }
+      break;
+    case "spotify-this-song":
+      if (argument !== "") {
+        spotifyThis(argument);
+      } else {
+        spotifyThis();
+      }
+      break;
+    case "movie-this":
+      if (argument !== "") {
+        movieThis(argument);
+      } else {
+        movieThis();
+      }
+      break;
+    case "do-what-it-says":
+    case "trust-me":
+      trustMe();
+      break;
+    default:
```

Comment:
Instead of asking if there is an argument on every case just call the function and do the check there since in both scenarios you are calling the function anyway, it will separate the concerns more appropriately.